Registry/RegistryParse.py

```diff
@@ -288,19 +346,19 @@ def hive_name(self):
     def first_hbin_offset(self):
         """
         Get the buffer offset of the first HBINBlock as an unsigned integer.
-        Note: always returns 0x1000 nothing else is known.
+        Note: always returns 0x1000, nothing else is possible.
         """
         return 0x1000
 
-    def last_hbin_offset(self):
```

Comment:
see comment below about api breakage, does it make sense to actually rename this (maybe this is explained in a commit?)